src/routes/EResourceEditRoute.js

```diff
@@ -7,16 +7,28 @@ import { stripesConnect } from '@folio/stripes/core';
 import { LoadingView } from '@folio/stripes/components';
 
 import { checkScope, collapseAllSections, expandAllSections } from '@folio/stripes-erm-components';
-import View from '../components/views/PCIForm';
+import PCIForm from '../components/views/PCIForm';
+import TitleForm from '../components/views/TitleForm';
 import NoPermissions from '../components/NoPermissions';
 import { urls, withSuppressFromDiscovery } from '../components/utilities';
+import { resourceClasses } from '../constants';
 
 class EResourceEditRoute extends React.Component {
   static manifest = Object.freeze({
+    resource: {
```

Comment:
Can we change the name to maybe be `eresource` instead. `resource` is a more generic name that is used all across stripes connect and it tends to interfere here.

Reply:
sure